datafeeder/datafeeder.properties

```diff
@@ -37,9 +37,33 @@ front-end.config.uri=file:${georchestra.datadir}/datafeeder/frontend-config.json
 
 datafeeder.publishing.geoserver.api-url=http://geoserver:8080/geoserver/rest
 datafeeder.publishing.geoserver.public-url=${scheme}://${domainName}/geoserver
+# Use this for HTTP basic authentication to geoserver api url:
+#datafeeder.publishing.geoserver.auth.type=basic
+#datafeeder.publishing.geoserver.auth.basic.username=geoserver_privileged_user
+#datafeeder.publishing.geoserver.auth.basic.password=gerlsSnFd6SmM
+# Use this for HTTP-headers based authentication to GeoServer's api url:
+datafeeder.publishing.geoserver.auth.type=headers
+datafeeder.publishing.geoserver.auth.headers.[sec-proxy]=true
+datafeeder.publishing.geoserver.auth.headers.[sec-username]=datafeeder-application
+datafeeder.publishing.geoserver.auth.headers.[sec-roles]=ROLE_ADMINISTRATOR
+
 
 datafeeder.publishing.geonetwork.api-url=http://geonetwork:8080/geonetwork
 datafeeder.publishing.geonetwork.public-url=${scheme}://${domainName}/geonetwork
+# Use this for HTTP basic authentication to Geonetwork's api url:
+#datafeeder.publishing.geonetwork.auth.type=basic
+#datafeeder.publishing.geonetwork.auth.basic.username=
+#datafeeder.publishing.geonetwork.auth.basic.password=
+# Use this for HTTP-headers based authentication to Geonetwork's api url:
+datafeeder.publishing.geonetwork.auth.type=headers
+datafeeder.publishing.geonetwork.auth.headers.[sec-proxy]=true
+datafeeder.publishing.geonetwork.auth.headers.[sec-username]=testadmin
```

Comment:
I would add a comment to indicate that the user has to be "fully resolved by GeoNetwork" (e.g. in case of using geOrchestra, having just the header won't be enough, as GN will do an extra LDAP connection to figure out who the user is)